@here/olp-sdk-dataservice-write/lib/model/StartBatchRequest.ts

```diff
@@ -0,0 +1,107 @@
+/*
+ * Copyright (C) 2020 HERE Europe B.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+/**
+ * @hidden Metadata about any dependencies on other catalogs that the publication may have.
+ * @todo use model from generated API after the API will be merged to the master.
+ */
+interface VersionDependency {
+    /**
+     * Indicates the type of data dependency. If this value is set to false, this dependency is an indirect dependency.
+     * This means that data from this catalog was not directly used by a data processing filter to update the current
+     * version of the catalog. Instead, the data was only used by upstream pipelines to generate the input data for
+     * a data processing pipeline.
+     */
+    direct?: boolean;
+
+    /**
+     * The HERE Resource Name (HRN) of the catalog that the publication depends on.
+     */
+    hrn?: string;
+
+    /**
+     * The version of the catalog metadata that the publication depends on.
+     */
+    version?: number;
```

Comment:
Are they all optional?

Reply:
yes, it's generated from spec.

Reply:
2checked. You are right, it should be required. Updated, thanks.